response_operations_ui/templates/messages.html

```diff
@@ -106,4 +106,4 @@ <h1 class="saturn u-mt-l">Oops! Something went wrong</h1>
     </div>
   </div>
 </div>
-{% endblock %}
+{% endblock %}
```

Comment:
Add newline

Reply:
@insacuri not sure if this a cosmetic thing or mandatory checkstyle. I mean it s html/flask page. I would recomend a checkstyle at python level.